content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -0,0 +1,569 @@
+# Running Experiments
+
+## Motivation
+
+Machine Learning and Data Science projects usually involve experimentation.
+Motivation of these experiments can range from finding good hyperparameters to
+checking data and concept drift. DVC 2 introduced a new set of commands to
+manage experiments with minimum boilerplate. It allows to run experiments
+defined by pipelines, track their associated data and model files, set
+parameters for each, push experiment parameters and code to Git remotes without
+committing them, create branches and persist them.
+
+In this part of the DVC User's Guide we will explain how DVC runs the
+experiments, setting parameters, running the experiments in parallel, or in
+temporary directories.
+
+If this is the first time you're introduced to Machine Learning experimentation,
+it may be better to start with the
+[Experiment's Trail](/doc/start/experiments/).
+
+## The Pipeline
+
+DVC uses pipelines composed of <abbr>stages</abbr>. Stages are isolated and
+interdependent steps that has a command, a set of dependencies, and output.
+Stages can depend onto each other and can build a <abbr>DAG</abbr>.
+
+DVC employs the pipeline to run the experiment commands. Each pipeline has an
+end-point, which is the last command in the pipeline. The output of this last
+command is considered the output of experiment. Additionally DVC tracks
+artifacts from intermediate stages, and doesn't reproduce them if their
+dependents have not changed.
+
+DVC pipelines are a detailed topic that we cover in [Get Started][gs-pipelines]
+and the [User's Guide][ug-pipeline-files]. Here we assume that there is already
+a pipeline defined in `dvc.yaml` file in the <abbr>project</abbr>.
+
+[gs-pipelines]: /doc/start/data-pipelines
+[ug-pipeline-files]: /doc/user-guide/project-structure/pipelines-files
```

Comment:
I'm not seeing a clear connection between needing to explaining pipelines and running experiments. It could just be a single word linked to https://dvc.org/doc/user-guide/project-structure/pipelines-files or https://dvc.org/doc/command-reference/dag . Clearly we need a separate guide about pipelines or at least a basic concept page for now, but that's out of scope.

p.s. We'll get to it so not much need to worry about it here and when we do we will update all these links too.

Reply:
I see the following section already has the link I was thinking about:

![image](https://user-images.githubusercontent.com/1477535/129416282-7604bcb7-c355-4772-8255-5a7bca75514d.png)

I think the guide can start there. If needed one short paragraph for background could be OK before that IMO.

Reply:
IMO this guide should be a bit self-contained. Without pipelines, the user cannot run the experiments. 

Learning/linking pipelines elsewhere is asking the reader to learn `dvc repro`, possibly `dvc run`, `dvc stage` and related commands before coming to `dvc exp`. This summary is aimed towards to "save" the user from those topics and focus only on "pipelines in experiments." Some parts of it may be a bit too much as @dberenbaum suggested, but linking other parts of the documentation causes reader to divert attention.

Reply:
What do you mean self-contained though?  The docs are self-contained but we don't need to cover every feature in every guide. There are 2 chapters in this guide before you even get to the topic (running experiments).

> linking pipelines elsewhere is asking the reader to learn dvc repro, possibly dvc run, dvc stage

No, why? Again, the fact that those other concepts don't have a guide yet doesn't mean we have to write them all at the same time. Let's focus on running exps for now. We'll get to writing about pipelines later, meanwhile the cmd ref is where we can link.

Reply:
By _self-contained_, I mean to cover the necessary part of the pipelines within the context of `exp run`. The pipelines section doesn't cover how to create or update them, for example, I think that's not relevant here. But _running_ pipelines is what _dvc exp run_ does, and without a proper summary here, the user is directed to all sorts of details. 

Also, to learn the pipelines itself, one needs to learn `dvc stage`, `dvc repro`, `dvc run`, etc. I would like the reader not to be distracted with other commands within the experiments guide.

Reply:
> running pipelines is what dvc exp run does

Makes sense but I still think we don't need the concept of pipelines here necessarily. Anyway, I'll try to finalize/merge this and open a separate issue to discuss/improve if needed ⏳

Reply:
Summarized in b88e34c5 for now but it's strange to see "running exps" as title and then it mainly talks about pipelines and running pipelines. confusing...